aboutus.js

```diff
@@ -0,0 +1,19 @@
+function getUserName() {
+  if (localStorage.userProfile) {
+    console.log(localStorage.userProfile);
```

Comment:
In most cases, at the point of 'Production Release' you would want to remove any non-critical logs from the code to ensure that it's clean and as small as possible.